cli/running/lua/launcher.lua

```diff
@@ -39,22 +39,156 @@ local function check_version(expected)
     return res
 end
 
+-- types of available options
```

Comment:
The sentence must start with a capital letter.

Reply:
Done